commands/namespaces_list.go

```diff
@@ -0,0 +1,76 @@
+package commands
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/spf13/cobra"
+)
+
+func init() {
+	// Setup flags that are used by multiple commands (variables defined in faas.go)
+	namespacesCmd.Flags().StringVarP(&gateway, "gateway", "g", defaultGateway, "Gateway URL starting with http(s)://")
+	namespacesCmd.Flags().BoolVar(&tlsInsecure, "tls-no-verify", false, "Disable TLS validation")
+	namespacesCmd.Flags().StringVarP(&token, "token", "k", "", "Pass a JWT token to use instead of basic auth")
+
+	faasCmd.AddCommand(namespacesCmd)
+	namespaceCmd.AddCommand(namespaceListCmd)
+}
+
+var namespacesCmd = &cobra.Command{
```

Comment:
Potentially make this one hidden? Because we want to have one going forward, it'll still work, but be less confusing (I hope)

Reply:
Done. It has been now hidden, as well marked deprecated